AlfrescoApp/View Controllers/FileFolder View Controller/BaseCollectionViewFlowLayout.m

```diff
@@ -19,15 +19,18 @@
 #import "BaseCollectionViewFlowLayout.h"
 #import "BaseLayoutAttributes.h"
 
+static CGFloat const itemSpacing = 10.0f;
```

Comment:
Might be better if this lived in the central constants class, otherwise it will be hard to find in the future.